### PR TITLE
Feature/show applications

### PR DESCRIPTION
**Pull request by rohan-naik07**
Hey! I have implemented two features.
  1) code . command for opening vs code.
  2) Show Applications panel with following features
      a) Show all applications 
      b) Search Applications 
      c) Frequent Applications used are displayed ( the data indicating frequency of each app is stored in local storage)

Issues to be solved->
  1) I have made the interface of the feature as Ubuntu like as possible. But still some minute changes are required such as size of 
      the bottom bars below "Frequent" and "All".
  2) I have noticed a bug that **when you try to open an application again after closing it , it does not open**. Please help me fix 
      it.

### CODE REVIEW

**Comment by vivek9patel** [comment]
great work!!!!
I need two things,
- which application is not opening after closing it? and from where are you opening it (like desktop, sidebar,...)?
- I'll solve the bottom bar issue this weekend

**Comment by rohan-naik07** [comment]
Any application.
if we open any app by double clicking from anywhere, it opens for first time. Then if we close it and try to open again, it doesn't open.

**Review by vivek9patel — COMMENTED** [review]
> Any application.
> if we open any app by double clicking from anywhere, it opens for first time. Then if we close it and try to open again, it doesn't open.

hmm, I'll try at my end to solve it!

File: src/components/apps/terminal.js [hunk 1]
```diff
@@ -255,7 +262,7 @@ export class Terminal extends Component {
                 result = "<img class=' w-2/5' src='./images/memes/used-sudo-command.jpg' />";
                 break;
             default:
-                result = "Command '" + main + "' not found, or not yet implemented.<br>Available Commands: [ cd, ls, echo, clear, exit, mkdir ]";
+                result = "Command '" + main + "' not found, or not yet implemented.<br>Available Commands: [ cd, ls, echo, clear, exit, mkdir,code .]";
```
[the remark this answers is not in the document]
Comment (vivek9patel) [reply src/components/apps/terminal.js:265]: can you push this change?

[the remark this answers is not in the document]
Comment (rohan-naik07) [reply src/components/apps/terminal.js:265]: Okay

**Pull request merged**